gladis/src/lib.rs

```diff
@@ -44,11 +44,39 @@ pub struct NotFoundError {
     pub typ: String,
 }
 
+impl Display for NotFoundError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "Identified {} of type {} was not found",
```

Comment:
typo: "Identified" -> "Identifier".
Also, do not use uppercase here, because this will lead `GladisError::fmt` to return strange case:

> Not found error: Identifier foo of type gtk::Label was not found

It should be "identifier" all in small caps.

Reply:
Should it be all lowercase? ([guidelines](https://rust-lang.github.io/api-guidelines/interoperability.html#error-types-are-meaningful-and-well-behaved-c-good-err)). Otherwise it would look strange if the error was wrapped by a consumer of the crate, i.e `Gladis error: Not found error: identifier foo of type gtk::Label was not found`.

So should it be `not found error: identifier foo of type gtk::Label was not found`

Reply:
All lowercase is better I think, to avoid this kind of composition issue.